### PR TITLE
feat(rbac): removed owner role and changed member permissions

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Description 📣

1. Removed owner
2. Wrote migration script to map owners to admin
3. Changed member name to developer in ui for project members
4. Updated project member permissions

# Tests 🛠️

```sh
# Here's some code block to paste some code snippets
```

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/code-of-conduct). 📝

### CODE REVIEW

File: backend/src/controllers/v3/secretsController.ts [hunk 1]
```diff
@@ -55,6 +55,9 @@ export const getSecretsRaw = async (req: Request, res: Response) => {
     secretPath = getFolderWithPathFromId(folder.nodes, folderId).folderPath;
   }
 
+  if (!environment || !workspaceId)
```
Comment (maidul98) [inline backend/src/controllers/v3/secretsController.ts:58]: good catch

File: backend/src/validation/secrets.ts [hunk 2]
```diff
@@ -225,8 +225,8 @@ export const GetSecretsV2 = z.object({
 
 export const GetSecretsRawV3 = z.object({
```
Comment (maidul98) [inline backend/src/validation/secrets.ts:226]: tested, it works

**Review by maidul98 — APPROVED** [review]
Reviewed, looks good to me

**Comment by zifeo** [comment]
@maidul98 what happens to the users which are owner when they switch to the new release?

**Comment by maidul98** [comment]
Owners will be migrated to to `admin` role. Did you have a specific concern?

**Comment by zifeo** [comment]
@maidul98 upgrading from 28 to latest does not seem to migrate, had to manually downgrade to admin for connection to work after the upgrade is operated

**Comment by maidul98** [comment]
@zifeo i found the issue. I'll push out a new release shorty

**Pull request merged**